parser-typechecker/src/Unison/PrintError.hs

```diff
@@ -1036,6 +1036,17 @@ prettyParseError s = \case
     "I expected a non-empty watch expression and not just \">\""
   go (Parser.UnknownAbilityConstructor tok _referents) = unknownConstructor "ability" tok
   go (Parser.UnknownDataConstructor    tok _referents) = unknownConstructor "data" tok
+  go (Parser.UnknownId               tok referents references) = Pr.lines
+    [ if missing then 
+        "I couldn't resolve the reference " <> style ErrorSite (HQ.toString (L.payload tok)) <> "."
+      else
+        "The reference " <> style ErrorSite (HQ.toString (L.payload tok)) <> " was ambiguous."
```

Comment:
Did you meant to print the referents / references here?

Reply:
I deliberately didn't bother but I think it could be nice! It's not too terrible as is, user can always `view` the identifier that's ambiguous.